extensions/wikia/WeiboTag/WeiboTagController.class.php

```diff
@@ -0,0 +1,76 @@
+<?php
+class WeiboTagController extends WikiaParserTagController {
+	const PARSER_TAG_NAME = 'weibo';
+	const TAG_SRC = 'http://widget.weibo.com/relationship/bulkfollow.php?';
+	const TAG_SOURCE_ALLOWED_PARAMS_WITH_DEFAULTS = [
+		'color' => '',
+		'count' => '',
+		'dpc' => '',
```

Comment:
We don't know what it does (seemingly nothing) so there is no way to document it. Should we remove it?